tests/kit/model_zoo/transformers/gpt.py

```diff
@@ -103,6 +130,7 @@ def date_gen_for_double_heads():
     hidden_dropout=0,
     problem_type="single_label_classification",
     pad_token_id=50256,
+    tie_word_embeddings=False,
```

Comment:
Origin GPT2 model ties embedding. Why do you modify this?

Reply:
Fixed